map/place_page_info.cpp

```diff
@@ -162,6 +162,11 @@ std::string Info::FormatSubtitle(bool withType) const
   if (GetWheelchairType() == ftraits::WheelchairAvailability::Yes)
     append(kWheelchairSymbol);
 
+  auto const fee = GetMetadata(feature::Metadata::FMD_FEE);
+  if (!fee.empty())
+    if (fee == "yes")
+      append("💰");
```

Comment:
It's not better to use [unicode number](https://symbl.cc/en/1F4B0/) like wheelchair icon?

Reply:
Changed to text